app/test_app.py

```diff
@@ -17,7 +17,7 @@ def test_get_sds_data_found():
         "/v1/unit_data",
         params={
             "dataset_id": "c067f6de-6d64-42b1-8b02-431a3486c178",
-            "identifier": "1234678912A",
+            "identifier": "1234678912",
```

Comment:
Should this be 12346789012? same below

Reply:
The test will still pass since its unused but for clarity?

Reply:
Update for clarity. We should consider updating the mock to check that value, at the moment the dataset is hardcoded but it doesn't need to be. Makes adding new schemas easier also.